src/test/java/org/example/integration/AuthIntegrationTest.java

```diff
@@ -0,0 +1,85 @@
+package org.example.integration;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.example.JobRoleManagerApplication;
+import org.example.JobRoleManagerConfiguration;
+import org.example.models.LoginRequest;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.Base64;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class AuthIntegrationTest {
+    private final String apiUrl;
+
+    private static final DropwizardAppExtension<JobRoleManagerConfiguration> APP =
+            new DropwizardAppExtension<>(JobRoleManagerApplication.class);
+
+    public AuthIntegrationTest() {
+        String apiUrl = System.getenv("API_URL");
+        if (apiUrl == null || apiUrl.isEmpty()) {
+            this.apiUrl = "http://localhost:8080/api/";
+        } else {
+            this.apiUrl = apiUrl;
+        }
```

Comment:
instead of having all of this logic, could we simplify this to be String apiUrl = System.getenv("API_URL") || "http://localhost:8080/api/";

Reply:
That would lead to an invalid java operation, in the case that System.getenv("API_URL") resolves to a string, as such we'll be comparing two strings which isn't allowed with the "||" operator. Although my code does look bulky, I can look to see if I can do further refactoring.